multiworld/envs/mujoco/sawyer_xyz/sawyer_reach.py

```diff
@@ -125,6 +125,12 @@ def _reset_hand(self):
             self.data.set_mocap_quat('mocap', np.array([1, 0, 1, 0]))
             self.do_simulation(None, self.frame_skip)
 
+    def reset(self):
```

Comment:
I'd just delete this now since it's the same as the base class (MujocoEnv) implementation